proto/api/v1/tenant.proto

```diff
@@ -83,6 +92,16 @@ message TenantServiceDeleteRequest {
   string login = 1;
 }
 
+message TenantServiceCheckAdmittedRequest {
```

Comment:
The User returned in the console should also contain the information if he/she is already admittet, but please check

Reply:
It did not, but this should be changed that no extra backend call is required to check for admittance after login, i will check

Reply:
OK:
- https://github.com/metal-stack-cloud/api/pull/36
- https://github.com/metal-stack-cloud/api-server/pull/182

Reply:
No i am stupid, you where right. If you fix the deleted stuff we can proceed